### PR TITLE
Add base structure for dockerfile

### PR DESCRIPTION
**Pull request by dbohomiakov**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+FROM python:3.5.6-alpine3.8
```
Comment (andriisoldatenko) [inline Dockerfile:1]: let's use Python 3.6/3.7 image.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ENV AIRFLOW_GPL_UNIDECODE yes
+ARG USER=airflow
+ARG AIRFLOW_HOME=/usr/local/${USER}
+RUN mkdir -p ${AIRFLOW_HOME}
+
+RUN adduser -D airflow
+RUN chown -R airflow: ${AIRFLOW_HOME}
+USER airflow
+
+RUN mkdir -p ${AIRFLOW_HOME}/dags
```
Comment (andriisoldatenko) [inline Dockerfile]: you can combine 2 lines
mkdir -p ${AIRFLOW_HOME}/dags {AIRFLOW_HOME}/sql

File: Dockerfile (hunk 1 above)
Comment (dbohomiakov) [reply Dockerfile:1]: airflow doesn't support py3.6 officially yet https://github.com/apache/incubator-airflow/pull/3816. Propose to add later when there will be official support.

Comment (andriisoldatenko) [reply Dockerfile:1]: :)

**Pull request merged**